ompi/tools/mpirun/Makefile.am

```diff
@@ -22,6 +23,11 @@ mpirun_SOURCES = \
 mpirun_LDADD = \
 	$(top_builddir)/opal/libopen-pal_core.la
 
+mpirun_CPPFLAGS = \
```

Comment:
I think this is safe from a build point of view, because the list of frameworks will not change based on any input to configure.  The only way to change the list of frameworks would be to re-run autogen.pl, which I think will always cause opal_config.h to be touched, which will cause main.c to be rebuilt.  But I'm only 99% certain.  I also think this is a corner case unlikely to be worth the time to resolve, but spent time thinking about it, so you get a comment.